pulsar-broker/src/main/java/org/apache/pulsar/broker/service/nonpersistent/NonPersistentTopic.java

```diff
@@ -748,6 +749,20 @@ public void updateRates(NamespaceStats nsStats, NamespaceBundleStats bundleStats
                 topicStatsStream.writePair("msgThroughputOut", subMsgThroughputOut);
                 topicStatsStream.writePair("msgRateRedeliver", subMsgRateRedeliver);
                 topicStatsStream.writePair("type", subscription.getTypeString());
+
+                // Write entry filter stats
+                Dispatcher dispatcher0 = subscription.getDispatcher();
+                if (null != dispatcher0) {
+                    topicStatsStream.writePair("entryFilterProcessedMsgs",
+                            dispatcher0.getFilterProcessedMsgCount());
+                    topicStatsStream.writePair("entryFilterAcceptedMsgs",
+                            dispatcher0.getFilterAcceptedMsgCount());
+                    topicStatsStream.writePair("entryFilterRejectedMsgs",
+                            dispatcher0.getFilterRejectedMsgCount());
+                    topicStatsStream.writePair("entryFilterRescheduledMsgs",
+                            dispatcher0.getFilterRescheduledMsgCount());
```

Comment:
We use `MsgCount` elsewhere, and it'd be great to make it consistent. Also, I don't think we need the `entry` filter prefix.  
```suggestion
                    topicStatsStream.writePair("filterProcessedMsgCount",
                            dispatcher0.getFilterProcessedMsgCount());
                    topicStatsStream.writePair("filterAcceptedMsgCount",
                            dispatcher0.getFilterAcceptedMsgCount());
                    topicStatsStream.writePair("filterRejectedMsgCount",
                            dispatcher0.getFilterRejectedMsgCount());
                    topicStatsStream.writePair("filterRescheduledMsgCount",
                            dispatcher0.getFilterRescheduledMsgCount());
```